neps/nep-0393.md

```diff
@@ -404,8 +404,17 @@ trait SBTRegistry {
     /// Must also emit `Burn` event if the SBT tokens are burned (removed).
     fn sbt_revoke(&mut self, tokens: Vec<TokenId>, burn: bool);
 
-    /// Similar to `sbt_revoke`, but revokes all `owner`s tokens issued by the caller.
-    fn sbt_revoke_by_owner(&mut self, owner: AccountId, burn: bool);
+    /// Similar to `sbt_revoke`. Allows SBT issuer to revoke all tokens by holder either by
+    /// burning or updating their expire time. The function will try to revoke
+    /// at most `MAX_REVOKE_PER_CALL` tokens (to fit into the tx gas limit),
```

Comment:
What is `MAX_REVOKE_PER_CALL`? It is the only reference here in this document. I believe it should be removed here.